### PR TITLE
doc qgroup consumer

### PR DESCRIPTION
**Pull request by aricart**
- [doc] added section in the doc about queue consumers
- [test] added a test for qgroup consumer using autoack

### CODE REVIEW

File: jetstream.md [hunk 1]
```diff
@@ -254,6 +254,28 @@ setInterval(() => {
 Note the above example is contrived, as the pull interval is fixed based on some
 interval.
 
+#### JetStream Queue Consumers
+
+Queue Consumers allow scaling the processing of messages stored in a stream. To
+create create a Queue Consumer, you have to set its `deliver_group` property to
```
Comment (kozlovic) [inline jetstream.md:260]: "create" is repeated.

**Review by kozlovic — CHANGES_REQUESTED** [review]
typo..

File: jetstream.md (hunk 1 above)
Comment (aricart) [reply jetstream.md]: fixed

File: jetstream.md [hunk 2]
```diff
@@ -254,6 +254,28 @@ setInterval(() => {
 Note the above example is contrived, as the pull interval is fixed based on some
 interval.
 
+#### JetStream Queue Consumers
+
+Queue Consumers allow scaling the processing of messages stored in a stream. To
+create a Queue Consumer, you have to set its `deliver_group` property to the
+name of the queue group (or use the `ConsumerOptsBuilder#queue()`. Then reuse
```
Comment (kozlovic) [inline jetstream.md:261]: I think you have a missing `)`, to close the "( or use the..".

Comment (aricart) [reply jetstream.md]: Thanks Ivan - my eyes are not focusing right.

**Review by kozlovic — APPROVED** [review]
LGTM

**Pull request merged**